customers/views.py

```diff
@@ -25,17 +26,8 @@ def create():
             email=email,
```

Comment:
is it redundant to define variables like `email = request_data["email"]` when you could do `email=request_data["email"]` and save 5 lines of code?

Reply:
You are absolutely right! Thank you!